### PR TITLE
Travis CI and Appveyor CI

### PR DESCRIPTION
**Pull request by cadecairos**
fixes https://github.com/MozillaFoundation/mofo-devops/issues/509

### CODE REVIEW

**Comment by mmmavis** [comment]
I'm gonna reassign this to @Pomax for review as I'm not familiar with this repo.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+python:
+  - "3.6"
+
+services: postgresql
+
+# command to install dependencies
+install: "pip install -r requirements.txt"
+
+before_script:
+  - psql -c "CREATE DATABASE pulseapi;" -U postgres
```
Comment (Pomax) [inline .travis.yml]: doesn't `migrate` already take care of this?

Comment (cadecairos) [inline .travis.yml]: Let's test!

**Comment by cadecairos** [comment]
@Pomax nope, we need to manually create the database. migrate only creates tables.

**Pull request merged**